src/Controller/Organisation/Create.php

```diff
@@ -14,7 +13,11 @@
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\RouterInterface;
 use Twig\Environment as Twig;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
```

Comment:
Any chance not to use the Sensio extra bundle and annotations here? Use the firewall config rather than annotations please.

Reply:
I will look deeper into it, but at first glance, there is no easy way to deal ROLE_ with URLs like Sponsorship/{id}/create in the firewall config

Reply:
Ok made the change, regex are possible, used { path: ^/organisations/*/edit, roles: ROLE_ADMIN }